test/test-tcp-fastopen.c

```diff
@@ -0,0 +1,247 @@
+/* Copyright Joyent, Inc. and other Node contributors. All rights reserved.
```

Comment:
Change the header to something like [this](https://github.com/libuv/libuv/blob/v1.x/test/test-fs-copyfile.c#L1)?